CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.8.0 (2018-05-10)
```

Comment:
I'll switch this to `Unreleased` here. We'd want to update this on master before pushing the gem, if possible, because otherwise the copy of `CHANGELOG.md` that is included with the gem will say that feature is `Unreleased`.